Mathlib/Algebra/Algebra/Spectrum.lean

```diff
@@ -141,6 +141,18 @@ theorem mem_resolventSet_iff {r : R} {a : A} : r ∈ resolventSet R a ↔ IsUnit
   Iff.rfl
 #align spectrum.mem_resolvent_set_iff spectrum.mem_resolventSet_iff
 
+theorem algebraMap_mem_iff (S : Type*) {R A : Type*} [CommSemiring R] [CommSemiring S]
```

Comment:
Should these 2 lemmas be `@[simp]`?

Reply:
Otherwise LGTM.

Reply:
@urkud I'm not sure, but we'll say yes for now.